static/elements/chromedash-metadata.html

```diff
@@ -55,7 +55,7 @@
       if (detail) {
         // Came from an internal click.
         this.selected = Polymer.dom(e).localTarget.dataset.version;
-        this.fire('milestoneselect', {version: this.selected});
+        this.fire('queryChanged', {version: this.selected});
```

Comment:
You'll also want to remove https://github.com/GoogleChrome/chromium-dashboard/blob/master/static/elements/chromedash-featurelist.html#L242.